tox.ini

```diff
@@ -34,6 +24,8 @@ setenv =
     clocale: LC_CTYPE = C.ascii
     clocale: LC_ALL = C
     devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scipy-wheels-nightly/simple
+    PYTHONHASHSEED = 1
```

Comment:
I think it is fine to have this to get this in, but can we raise an issue that it should be removed in the future? I do not think we should have any tests that depend on the hash seed (e.g., such that running tests without `tox` continues to work).

Reply:
This seems to be the only issue left - to perhaps add a comment here why this line is present, and raise an issue to (eventually) remove it.

Otherwise, this definitely is ready to go in! (indeed, I thought it had long been merged!)

Reply:
Ah, I don't think this is needed at all, this should be set by tox in the latest 4.0.x version

Reply:
https://github.com/tox-dev/tox/commit/5cef03092cc06fd598d66a821751cdb1c4789f1c

Reply:
Super, then indeed ready to merge!

Reply:
p.s. And at some level good to see that the problems were really mostly with `tox`, not with us! As you noted, all that's really left is a change of spaces to commas.